src/components/Book.js

```diff
@@ -1,61 +1,47 @@
-import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
-import { checkStatus, removeBook } from '../redux/books/booksSlice';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import booksDisplay from './displayBooks';
 import Form from './Form';
-import books from './books';
+import removeBook from './removeBook';
 
-const Books = () => (
-  <div>
-    {books.map((book) => (
-      <Book title={book.title} category={book.category} author={book.author} progress="64%" chapter="Chapter 17" id={book.id} key={book.id} />
-    ))}
-    <Form />
-  </div>
-);
-
-const Book = ({
-  category, title, author, progress, chapter, id,
-}) => {
+const Books = () => {
   const dispatch = useDispatch();
+  const books = useSelector((state) => (state.books.books));
+  useEffect(() => {
+    dispatch(booksDisplay());
+  }, [books]);
 
   return (
-    <div className="book">
-      <div className="book-details">
-        <p>{category}</p>
-        <h2 className="title">{title}</h2>
-        <p className="author">{author}</p>
-        <ul className="book-action">
-          <li>Comments</li>
-          <button type="button" onClick={() => dispatch(removeBook({ id }))}>Remove</button>
-          <li>Edit</li>
-          <button type="button" className="status-btn" onClick={() => dispatch(checkStatus({ id }))}>Status</button>
-        </ul>
-      </div>
-      <div>
-        <div className="oval" />
-        <div>
-          <h3 className="per-completed">{progress}</h3>
-          <p className="completed">completed</p>
+    <div>
+      {books.map((book) => (
+        <div className="book" key={book.key}>
+          <div className="book-details">
+            <p>{book.category}</p>
+            <h2 className="title">{book.title}</h2>
+            <p className="author">{book.author}</p>
+            <ul className="book-action">
+              <li>Comments</li>
+              <li><button type="button" id={book.key} onClick={() => dispatch(removeBook(book.key))}>Remove</button></li>
+              <li>Edit</li>
+            </ul>
+          </div>
+          <div>
+            <div className="oval" />
+            <div>
+              <h3 className="per-completed">0%</h3>
+              <p className="completed">completed</p>
+            </div>
+          </div>
+          <div>
+            <p className="current-chapter">CURRENT CHAPTER</p>
+            <p className="chapter">Chapter</p>
+            <button type="button" className="progress-update">UPDATE PROGRESS</button>
```

Comment:
You have done a good job unfortunately, there are some required changes to make your app match the given design:

- [ ] Kindly add the correct font families, font size, and font weight for all highlighted text in the image below. and remove the links styles for the nav bar links.
- [ ] Kindly add the correct spacing using the correct margins to make your element spacing match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/flemton/bookstore/assets/68796232/033020cd-29f0-4c3e-9008-4e6698df6a2b)  | ![image](https://github.com/flemton/bookstore/assets/68796232/399fa41a-b84e-4f1e-b343-df1381338a0b)  |